src/img/icons.json

```diff
@@ -48,7 +48,12 @@
     "tag",
     "three-dots-row",
     "unlock",
-    "wallet"
+    "wallet",
+    "expenditure",
```

Comment:
Please sort all the new icons added to this list alphabetically. 

Sadly, this has to be done manually.